knx/dpt/types_18.go

```diff
@@ -0,0 +1,40 @@
+package dpt
+
+import "fmt"
+
+// DPT_18001 represents DPT 18.001 / Scene Control.
+type DPT_18001 uint8
+
+func (d DPT_18001) Pack() []byte {
+	if d <= 63 || (d >= 128 && d <= 191) {
+		return packU8(uint8(d))
+	} else {
+		return packU8(63)
+	}
+}
+
+func (d *DPT_18001) Unpack(data []byte) error {
+	var value uint8
+
+	if err := unpackU8(data, &value); err != nil {
+		return err
+	}
+
+	if *d <= 63 || (*d >= 128 && *d <= 191) {
```

Comment:
This also compares against `d` and should not do that.

Reply:
Same for this.